src/components/qas/questions/qaMultiSelect/qaMultiSelect.module.scss

```diff
@@ -0,0 +1,69 @@
+:global {
+  @include react-md-menus-media;
+  @include react-md-selection-controls;
+
+  .qa-select {
+    label,
+    .label-pre,
+    .label-post {
+      @include copySecondary;
+      line-height: $baseLineHeight;
+    }
+
+    select {
+      @include copyPrimary;
+    }
+
+    &:not(.answerable) {
+      .select-wrapper::after {
+        border-top-color: $neutral20;
+      }
+
+      select,
+      label,
+      .label-pre,
+      .label-post {
+        color: $neutral20;
+      }
+
+      select {
+        &:focus,
+        &:hover {
+          + hr {
+            max-height: 1px;
+            background-color: $neutral20;
+          }
+        }
+      }
+
+      hr {
+        max-height: 1px;
+        background-color: $neutral20;
+      }
+    }
+
+    &.answerable {
+      label,
+      .label-pre {
+        color: $basePrimary;
+      }
+    }
+  }
+}
+
+.multi-select-dropdown {
+  position: relative;
+
+  :global .md-list--menu {
+    position: absolute;
+    z-index: 1;
+  }
+
+  .selected-options {
+    height: 20px;
+    font-weight: 200;
+    color: #222222;
```

Comment:
I prefer that as much as possible colors are sass variables so they're easier to keep track of and we don't let the project start using dozens of almost the same colors, so let's use $black instead of #222222 in both of these cases here